model_hub/model_hub/mmdetection/_trial.py

```diff
@@ -81,7 +91,6 @@ def __init__(self, context: det_torch.PyTorchTrialContext) -> None:
             mmcv.runner.build_optimizer(self.model, self.cfg.optimizer)
         )
         self.model.zero_grad()
-        self.optimizer.zero_grad()
```

Comment:
Is it ok to just remove this?

Reply:
optimizer parameters are a subset of model parameters so model.zero_grad() will cover everything.